addons/udes_priorities/models/priority.py

```diff
@@ -26,17 +26,36 @@ class UdesPriority(models.Model):
         default=lambda self: uuid4(),
     )
 
+    @api.multi
+    @api.depends("reference")
+    def _compute_outstanding_pickings(self):
```

Comment:
Nit: Can we add a doc-string? Doesn't need to be super detailed, but is good practice.